src/components/button/Button.tsx

```diff
@@ -0,0 +1,93 @@
+import * as React from 'react';
+import classNames from 'classnames';
+import LoadingIndicator from '../loading-indicator';
+import RadarAnimation from '../radar';
+
+export enum ButtonType {
+    BUTTON = 'button',
+    RESET = 'reset',
+    SUBMIT = 'submit',
+}
+export interface ButtonProps {
```

Comment:
Does this leave the interface open to having other properties [merged](https://www.typescriptlang.org/docs/handbook/declaration-merging.html#merging-interfaces) into it? Is that desirable? If not, should we use `type`, instead?

Reply:
Did we figure this out, @priyajeet?